src/markdown-pages/collect-data/opentelemetry/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Instrument and Analyze OpenTelemetry data'
+template: 'LabOverviewTemplate'
+description: 'Instrument and Analyze OpenTelemetry data'
+---
+
+<Intro>
+
+You're a developer who recently released a simple .NET weather forecast application to the world, and people can't stop talking about it! Surprised by the spike in popularity, you decided you should start collecting metrics and traces from your application so you can analyze them in New Relic.
```

Comment:
Is it necessary to mention "simple"? 
It reads to me like this guide will only work with simple applications and not the complicated ones.